analysis/online/BeamMonitorOnlineSpectra.cxx

```diff
@@ -122,6 +122,7 @@ InitStatus BeamMonitorOnlineSpectra::Init()
 
     run->AddObject(bmFold);
 
+    run->GetHttpServer()->RegisterCommand("Reset_BM_Histos", Form("/Objects/%s/->Reset_Histo()", GetName()));
```

Comment:
why is this duplicated? meant to be snapshot?

Reply:
Yes, meant to be snapshot